enhancements/machine-api/control-plane-machinesets.md

```diff
@@ -0,0 +1,232 @@
+---
+title: control-plane-machinesets
+authors:
+  - "@michaelgugino"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-01-19
+last-updated: 2021-01-19
+status: implementable
+---
+
+# Control Plane MachineSets
+
+<!-- toc -->
+- [Release Signoff Checklist](#release-signoff-checklist)
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [Story 1](#story-1)
+    - [Story 2](#story-2)
+  - [Implementation Details/Notes/Constraints [optional]](#implementation-detailsnotesconstraints-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+    - [Risk: Users might delete these new machinesets](#risk-users-might-delete-these-new-machinesets)
+    - [Risk: Users might delete too many control plane hosts carelessly](#risk-users-might-delete-too-many-control-plane-hosts-carelessly)
+    - [Risk: Conflicts with user-created operators or automation](#risk-conflicts-with-user-created-operators-or-automation)
+    - [Risk: Some clusters won't successfully automatically adopt machines](#risk-some-clusters-wont-successfully-automatically-adopt-machines)
+    - [Risk: Some users might scale the machinesets](#risk-some-users-might-scale-the-machinesets)
+- [Design Details](#design-details)
+  - [machine-api API modifications](#machine-api-api-modifications)
+  - [New Controllers](#new-controllers)
+  - [MachineSet deletion webhooks](#machineset-deletion-webhooks)
+  - [Open Questions [optional]](#open-questions-optional)
+    - [Should we modify the installer?](#should-we-modify-the-installer)
+  - [Test Plan](#test-plan)
+  - [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement documents the intent to adopt existing control plane machines
+into machinesets for all clusters utilizing the machine-api.
+
+## Motivation
+
+Currently, control plane hosts are deployed as individual machines without
+a machineset or other controller to assist in replacement operations.
+
+### Goals
+
+1. Adopt existing control plane machines into machinesets
+1. Add logic to ensure users don't break themselves with the new machinesets
```

Comment:
the list numbering isn't right :)

Reply:
that is using [markdown auto-numbering ](https://www.markdownguide.org/basic-syntax/#ordered-lists)

Reply:
👍